Makefile.am

```diff
@@ -808,3 +812,6 @@ coverage-clean:
 
 czech: check
 	@echo "This should do something much cooler"
+
+collabora-online-sbom.spdx.json: coolforkit-ns
```

Comment:
Nit: also depend on the python script to help re-running this when you touch it?